util/request_source.go

```diff
@@ -36,6 +36,15 @@ const (
 	SourceUnknown = "unknown"
 )
 
+// explicit source types.
+const (
+	ExplicitTypeEmpty = ""
+)
+
+// ExplicitTypeList is the list of all explicit source types.
+// this hack way to FIX cse tests
+var ExplicitTypeList = []string{ExplicitTypeEmpty}
```

Comment:
CSE branch will not relay by tidbcloud after the `region-client` transfer out. so for making the test pass, I added an empty list here.